src/api/app/websocket.ts

```diff
@@ -426,14 +431,7 @@ const callZomeTransform: Transformer<
     if (hostSigner) {
       return hostSigner.signZomeCall(request);
     } else {
-      const env = getLauncherEnvironment();
-      if (!env) {
-        return signZomeCall(request);
-      }
-      if (env.FRAMEWORK === "electron") {
-        return signZomeCallElectron(request);
-      }
-      return signZomeCallTauri(request);
+      return signZomeCall(request);
```

Comment:
Why is zome call singing by the runtime being removed?

Reply:
Only the legacy methods are removed. The framework agnostic `hostSigner.signZomeCall()` is still there.

Reply:
Ahh I see what you're saying, nice okay!